common/rpm/containers-common.spec

```diff
@@ -184,7 +145,6 @@ ln -s ../../../..%{_sysconfdir}/pki/entitlement %{buildroot}%{_datadir}/rhel/sec
 ln -s ../../../..%{_sysconfdir}/rhsm %{buildroot}%{_datadir}/rhel/secrets/rhsm
 ln -s ../../../..%{_sysconfdir}/yum.repos.d/redhat.repo %{buildroot}%{_datadir}/rhel/secrets/redhat.repo
 
-# Placeholder check to silence rpmlint warnings
```

Comment:
Also worth preserving? Or drop the section entirely if it is not applicable.

Reply:
ah, my bad. Adding back..